app/ethermint.go

```diff
@@ -147,8 +148,8 @@ func NewEthermintApp(logger tmlog.Logger, db dbm.DB, loadLatest bool,
 		keyGov:         sdk.NewKVStoreKey(gov.StoreKey),
 		keyParams:      sdk.NewKVStoreKey(params.StoreKey),
 		tkeyParams:     sdk.NewTransientStoreKey(params.TStoreKey),
-		evmStoreKey: sdk.NewKVStoreKey(params.StoreKey),
-		evmCodeKey: sdk.NewKVStoreKey(params.StoreKey),
+		evmStoreKey: sdk.NewKVStoreKey(evmtypes.EvmStoreKey),
```

Comment:
Formatting could be aligned but I'm fine with this since it will be changed later